content/md/imports-exports/quick-export.md

```diff
@@ -44,7 +44,7 @@ Only the working channel is taken into account, and thus all the locale informat
 For this export from the grid, the completeness, the product status, their classification are not taken into account to determine what information can be exported. All the products and product models selected in the product grid, regardless of their details, are exported to the CSV file or XLSX file.
 
 ::: warning
-Note that the following columns are not exported during a Grid Context Export: Labels, Image, Completeness, "Created at", "Updated at" and Variant products.
+Note that the following columns are not exported: Labels, Image, Completeness, "Created at", "Updated at" and Variant products.
```

Comment:
Typos on column names:
it's Label (not Labels), and it's Complete (not Completeness) and maybe remove the " " for Created at and Updated at.
Thanks!

Reply:
When we name part of the interface, here columns names, it's better to use ` around the words.

Reply:
Like this:
```
`Labels`, `Image`, `Complete`,...
```

Reply:
Hey don't be so hard girls, Karen will no longer make PR ;)